code/game/objects/items/cards_ids.dm

```diff
@@ -1437,6 +1443,9 @@
 		theft_target = WEAKREF(interacting_with)
 		ui_interact(user)
 		return ITEM_INTERACT_SUCCESS
+	if(iscash(interacting_with))
+		insert_money(interacting_with, user)
+		return ITEM_INTERACT_SUCCESS
```

Comment:
probably better to make sure this `interact_with_atom(...)` proc calls the parent, rather than having the same snippet of code as the parent but again

Reply:
as it's not shown directly on the snippet, this is the `/obj/item/card/id/advanced/chameleon/interact_with_atom(...)` part.

Reply:
> probably better to make sure this `interact_with_atom(...)` proc calls the parent, rather than having the same snippet of code as the parent but again

I tried, but it didn't call, because here proc is rewriting.
`/obj/item/card/id/advanced/chameleon/interact_with_atom(atom/interacting_with, mob/living/user, list/modifiers)`

Reply:
Ah, we should be able to use `..()` here.
So say, having for example the following code _should_ call our parent proc:
```dm
/obj/item/card/id/advanced/chameleon/interact_with_atom(atom/interacting_with, mob/living/user, list/modifiers)
	if(isidcard(interacting_with))
		theft_target = WEAKREF(interacting_with)
		ui_interact(user)
		return ITEM_INTERACT_SUCCESS
	return ..()
```
Where it first runs `/obj/item/card/id/advanced/chameleon/interact_with_atom(...)`, then if it isn't an idcard returns `..()`, and thus calls and returns `/obj/item/card/id/interact_with_atom(...)` and handles our cash insertion code.

Reply:
It works, thank you!